vendor/milesight-iot/em300-th.yaml

```diff
@@ -2,14 +2,16 @@ name: EM300-TH
 description: The Milesight LoRaWAN® EM300-TH consists of a temperature and a humidity sensor monitoring indoor and outdoor environments. It is suitable for applications such as greenhouses, cold chains, food prevention, warehouses, residences, offices, and labs.
 
 hardwareVersions:
-  - version: 'V2.2'
+  - version: '2.x'
     numeric: 1
+  - version: '3.x'
+    numeric: 2
 
 firmwareVersions:
-  - version: '1.15'
+  - version: '1.22'
```

Comment:
Existing devices provisioned with these versions will not be able to load the right codec. Is this change really necessary?

Reply:
I rolled back the modification on the version number of firmware, and new version of hardware should remain due to updated feature with the new machine.